custom_components/tuya_local/devices/solar_inverter_gtb_series.yaml

```diff
@@ -0,0 +1,161 @@
+name: Smart Inverter GTB Series
+products:
+  - id: bf27badb1abd9bfc5beeuq
+    name: GTB Series
+primary_entity:
+  entity: sensor
+  class: energy
+  dps:
+    - id: 2
+      type: integer
+      name: sensor
+      unit: kWh
+      class: total_increasing
+      mapping:
+        - scale: 100
+    - id: 15
+      name: model
+      type: string
+    - id: 16
+      name: inverter_id
+      type: string
+secondary_entities:
+  - entity: sensor
+    class: power
+    category: diagnostic
+    dps:
+      - id: 10
+        type: integer
+        name: sensor
+        unit: W
+        force: true
+        class: measurement
+        mapping:
```

Comment:
There is no need to include a mapping for a scale of 1 (no scaling). But if you copied that from the tuya docs for the device, it should probably be `scale: 10`, as tuya scales are number of decimal places to shift rather than the divisors that this integration uses.